site/content/en/docs/tasks/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -0,0 +1,611 @@
+---
+title: "Pending Workloads on-demand"
+date: 2023-12-04
+weight: 3
+description: >
+---
+
+This page shows you how to monitor pending workloads with VisibilityOnDemand feature.
+
+The intended audience for this page are [batch administrators](/docs/tasks#batch-administrator), and [batch users](/docs/tasks#batch-user) for [Local Queue Visibiility section](#local-queue-visibility).
+
+From version v0.6.0, Kueue provides the ability for a batch administrators to monitor
+the pipeline of pending jobs, and help users to estimate when their jobs will
+start.
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+- [Kueue is installed](/docs/installation) in version v0.6.0 or later.
+- You have sufficient permissions according to [Setup RBAC section](#setup-rbac)
+
+### Enabling feature VisibilityOnDemand
+
+VisibilityOnDemand is an `Alpha` feature disabled by default, check the [Change the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) section of the [Installation](/docs/installation/) for details.
+
+## Monitor pending workloads on demand
+
+> _Available in Kueue v0.6.0 and later_
+
+To install a simple setup of cluster queue
+
+{{< include "examples/admin/single-clusterqueue-setup.yaml" "yaml" >}}
+
+
+run the following command:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/examples/admin/single-clusterqueue-setup.yaml
```

Comment:
would an URL like this work?
```
kubectl apply --server-side -k "github.com/kubernetes-sigs/kueue/config/default?ref=main"
```
(from our installation guide).

Reply:
IIUC this URL installs Kueue on a cluster, and does not create a ClusterQueue so I don't think they are equivalent

Reply:
What I mean is the overall shape of the URL (it doesn't have raw). Another alternative would be to point to the website itself, similar to this `https://kueue.sigs.k8s.io/examples/jobs/sample-job.yaml`.

Reply:
Oh yes, these URLs work, I've changed it